CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ set(CMAKE_DEBUG_POSTFIX D)
 set(CPPdefinitions)
 
 option(FMT_SEPARATE_COMPILATION "build fmt lib too" ${PROJECT_IS_TOP_LEVEL})
-option(FMT_USE_MODULES "Export a CXX_MODULE asio if possible" ${PROJECT_IS_TOP_LEVEL})
-option(FMT_IMPORT_STD "Import the all std header needed" NO)
+#XXX option(FMT_USE_MODULES "Export a CXX_MODULE asio if possible" ${PROJECT_IS_TOP_LEVEL})
```

Comment:
The alleged module name seems to be copy pasta.

Reply:
fixed